example_sidecars/label-sidecar/cmd/label-sidecar.go

```diff
@@ -0,0 +1,53 @@
+package main
+
+import (
+	flag "github.com/spf13/pflag"
+	corev1 "k8s.io/api/core/v1"
+	v12 "k8s.io/apiserver/pkg/quota/v1"
+	"k8s.io/kubernetes/pkg/quota/v1/evaluator/core"
+	"k8s.io/utils/clock"
+	"kubevirt.io/application-aware-quota-api/libsidecar"
+)
+
+const (
+	// Double Calculate double amount of usage for pod
+	Double configName = "double"
+	// Triple Calculate triple amount of usage for pod
+	Triple        configName = "triple"
+	labelAppLabel            = "label-app"
+)
+
+type configName string
+
+type labelCalculator struct {
+	evaluatorConfig configName
+}
+
+func (lc *labelCalculator) PodUsageFunc(podToEvaluate *corev1.Pod, _ []*corev1.Pod) (corev1.ResourceList, error, bool) {
```

Comment:
It's feels a bit odd to have the error return param at the end. We can change the signature to the following in a follow-up PR:
`PodUsageFunc(podToEvaluate *corev1.Pod, _ []*corev1.Pod) (corev1.ResourceList, bool, error)`

Another alternative is to provide a known error value to specify that the pod is not affected, therefore removing the bool argument:
```go
func (lc *labelCalculator) PodUsageFunc(podToEvaluate *corev1.Pod, _ []*corev1.Pod) (corev1.ResourceList, error) {
	if something {
		return nil, libsidecar.errPodNotAffected
	}
}
```

Reply:
Yes, you're absolutely right about that one. I will make sure to add another PR before this one gets merged.

Reply:
Here:
https://github.com/kubevirt/application-aware-quota/pull/56#issuecomment-2081421947

Reply:
Done :)